woudc_data_registry/epicentre/instrument.py

```diff
@@ -67,18 +69,23 @@ def build_instrument(ecsv):
     serial = str(ecsv.extcsv['INSTRUMENT']['Number'])
     station = str(ecsv.extcsv['PLATFORM']['ID'])
     dataset = ecsv.extcsv['CONTENT']['Category']
+    contributor = str(ecsv.extcsv['CONTRIBUTOR']['ID'])
```

Comment:
Trying to refer to a `#CONTRIBUTOR.ID` field will give a KeyError: the contributor ID field is `#DATA_GENERATION.Agency`.

See any of the Extended CSV files in https://woudc.org/archive/Archive-NewFormat/ for examples on the file format and core tables and fields.